pkg/cmd/clients.go

```diff
@@ -17,28 +17,32 @@ package cmd
 import (
 	"fmt"
 	"io"
-	"strings"
-
-	"github.com/aerogear/mobile-cli/pkg/cmd/input"
 	"github.com/aerogear/mobile-cli/pkg/cmd/output"
 	"github.com/aerogear/mobile-crd-client/pkg/apis/mobile/v1alpha1"
+	"github.com/aerogear/mobile-crd-client/pkg/client/servicecatalog/clientset/versioned"
 	mobile "github.com/aerogear/mobile-crd-client/pkg/client/mobile/clientset/versioned"
 	"github.com/olekukonko/tablewriter"
 	"github.com/pkg/errors"
-	"github.com/satori/go.uuid"
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"encoding/json"
```

Comment:
I think its good practice to list imports from stdlib first followed by external dependencies but this is just a nit.